c++/src/kj/compat/http.h

```diff
@@ -717,22 +717,18 @@ class HttpClient {
   // `url` and `headers` need only remain valid until `openWebSocket()` returns (they can be
   // stack-allocated).
 
-  struct ConnectResponse {
+  struct ConnectRequest {
     struct Status {
       uint statusCode;
       kj::StringPtr statusText;
       const HttpHeaders* headers;
-      kj::Maybe<kj::Own<kj::AsyncInputStream>> errorBody;
```

Comment:
Really because it's not really necessary and removing it simplifies some things (such as attaching the `ConnectionCounter` elsewhere.

Reply:
However, looking at the rest of the comments here... we might indeed need to bring it back. We'll need to think about how that impacts things that need to be *attached* to the streams. `ConnectionCounter`, for example. I presume that we'd have to transition that into a `Refcounted` so that we can add a reference to both the status promise (to be attached to `errorBody` if/once that exists) and another to `connection`. It gets a bit convoluted in here. Trying to work through this case is the main reason I wanted to see if we could get by without having `errorBody` at all.